go.mod

```diff
@@ -67,7 +67,9 @@ require (
 	github.com/weaveworks/common v0.0.0-20200201141823-27e183090ab1
 	go.etcd.io/bbolt v1.3.3
 	go.etcd.io/etcd v0.0.0-20190709142735-eb7dd97135a5
+	go.uber.org/atomic v1.5.0
```

Comment:
Instead of this additional dependency, can we use https://golang.org/pkg/sync/atomic/ for atomic loads and stores?

Reply:
We already have this dependency, although under old name (github.com/uber-go/atomic). I've removed old version in some PR #2127.